development/client/src/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+import { ApolloClient, HttpLink, InMemoryCache, ApolloProvider, makeReference } from '@apollo/client';
+
+const client = new ApolloClient({
+  cache: new InMemoryCache({
+    typePolicies: {
+      Color: {
+        keyFields: ["hex"],
+        fields: {
+          saved: {
+            read(_, { readField }) {
+              const hex = readField("hex");
+              const favoritedColors = readField("favoritedColors", makeReference("ROOT_QUERY"));
+              return favoritedColors.some(colorRef => {
+                return hex === readField("hex", colorRef);
+              });
+            }
+          },
+        }
+      }
+    },
+  }),
+  link: new HttpLink({
+    uri: 'http://localhost:4000',
+  }),
```

Comment:
With AC3 this can now just be

```suggestion
  uri: 'http://localhost:4000',
```

 (the link is automagically created behind the scenes)